benches/default_run.rs

```diff
@@ -0,0 +1,29 @@
+use criterion::{black_box, BenchmarkId, Criterion};
+use fst::Set;
+
+use meilisearch_tokenizer::{Analyzer, AnalyzerConfig, Token};
+
+pub fn criterion_benchmark(c: &mut Criterion, data_set: &[(&str, &str)]) {
+    let stop_words = Set::default();
+    let analyzer = Analyzer::new(AnalyzerConfig::default_with_stopwords(&stop_words));
+
+    // analyze a first time each text to trigger lazy initializations
+    for &(_name, text) in data_set {
+        analyzer.analyze(text);
+    }
+
+    let mut group = c.benchmark_group("default-run");
+
+    for &(name, text) in data_set {
+        group.bench_with_input(BenchmarkId::new("default-run", name), &(&analyzer, black_box(text)), |b, &(a, s)| b.iter(|| run(a, s)));
```

Comment:
what exactly is the difference between `bench_with_input` and simply capturing `(&analyzer, black_box(text))` in a `bench_function`?

Reply:
what do you mean by "capturing in a `bench_function`"?

Reply:
I mean you are basically using a closure here: `|b, &(a, s)| b.iter(|| run(a, s)` and passing (a, s) to it, why not directly do this: 
```suggestion
        group.bench_with_function(BenchmarkId::new("default-run", name), |b| b.iter(|| run(&analyzer, black_box(text)));
```

Reply:
don't know, I followed the `tutorial`

Reply:
Ok the difference is that `bench_with_input use the` black_box()` function on the input before running the benchmark